src/libmongoc/src/mongoc/mcd-time.h

```diff
@@ -62,56 +74,117 @@ mcd_now (void)
    return (mcd_time_point){._rep = bson_get_monotonic_time ()};
 }
 
-/// Create a duration from a number of microseconds
+/**
+ * @brief Create a duration from a number of microseconds.
+ *
+ * @param s A number of microseconds
+ * @return mcd_duration A duration corresponding to 's' microseconds.
+ *
+ * @note Saturates to the min/max duration if the duration is too great in
+ * magnitude.
```

Comment:
Consider removing the note. It is not necessary if there is no possibility of overflow.

Reply:
For the current impl it won't overflow, but I want to avoid making the current impl (as amicrosecond counter) part of the API guarantees.